sym/ed551a6a-f80e-4700-861b-a037225be664/symbol.lp

```diff
@@ -0,0 +1,96 @@
+(librepcb_symbol ed551a6a-f80e-4700-861b-a037225be664
+ (name "TCA9548A")
+ (description "Low-Voltage 8-Channel I2C Switch with Reset")
+ (keywords "i2c,mux,multiplexer")
+ (author "Marc Wittwer")
+ (version "0.1")
+ (created 2019-05-31T11:16:46Z)
+ (deprecated false)
+ (category e96d0dda-3100-4e68-8c33-e2a5bc80428b)
+ (pin 4f8991f6-3ea6-466b-8d9e-2351cf57cc8f (name "SD2")
+  (position 10.16 7.62) (rotation 180.0) (length 2.54)
+ )
+ (pin 6d06642e-f915-441e-ad9c-f43e835dccef (name "SC2")
+  (position 10.16 5.08) (rotation 180.0) (length 2.54)
+ )
+ (pin 4c9701bd-2037-4916-b1fe-be5955307bbd (name "SD3")
+  (position 10.16 2.54) (rotation 180.0) (length 2.54)
+ )
+ (pin 2fb196c1-33dd-4f1b-b320-fc31babf5b0c (name "SC3")
+  (position 10.16 0.0) (rotation 180.0) (length 2.54)
+ )
+ (pin db1902d6-c385-46b7-83ff-bfc81804b571 (name "SD4")
+  (position 10.16 -2.54) (rotation 180.0) (length 2.54)
+ )
+ (pin b5aeb8e6-0476-44c6-b9a2-1e92813ff727 (name "SC4")
+  (position 10.16 -5.08) (rotation 180.0) (length 2.54)
+ )
+ (pin 869712c0-afef-4423-bdff-5bbed716f7c7 (name "SD5")
+  (position 10.16 -7.62) (rotation 180.0) (length 2.54)
+ )
+ (pin 013ff497-0933-4b9f-9552-da6caaec36f2 (name "SC5")
+  (position 10.16 -10.16) (rotation 180.0) (length 2.54)
+ )
+ (pin 3180294c-ed90-4636-a55d-24f5ecdb5377 (name "SD6")
+  (position 10.16 -12.7) (rotation 180.0) (length 2.54)
+ )
+ (pin b41a7196-2c0b-4c76-9d55-f284b6449f43 (name "SC6")
+  (position 10.16 -15.24) (rotation 180.0) (length 2.54)
+ )
+ (pin 02711102-53f9-4134-a8ee-02c4435f1e17 (name "SD7")
+  (position 10.16 -17.78) (rotation 180.0) (length 2.54)
+ )
+ (pin d2e0f1e7-828c-4ea6-a9c8-48427e8543fa (name "SC7")
+  (position 10.16 -20.32) (rotation 180.0) (length 2.54)
+ )
+ (pin 8ea3408f-1695-4ea1-abcb-86d79ad7bafb (name "VIN")
+  (position -10.16 15.24) (rotation 0.0) (length 2.54)
+ )
```

Comment:
I would put VIN above GND, and increase the space between VIN and GND from 2.54 to 5.08 mm, so that a capacitor could be placed directly across them. See https://docs.librepcb.org/#libraryconventions-symbols-pinplacement